docs/user-guide.md

```diff
@@ -0,0 +1,856 @@
+    library and efficiently exposes the expressiveness of Python to the
+    browser.</p>
+    <p>Both Python interpreters supported by PyScript implement the same 
+    FFI to bridge the gap between the worlds of Python and the browser.</p>
+    </dd>
+
+    <dt><em>AI and Data science built in</em></dt>
+    <dd>Python is famous for its extraordinary usefulness in artificial
+    intelligence and data science. The Pyodide interpreter comes with many of
+    the libraries you need for this sort of work already baked in.</dd>
+
+    <dt><em>Mobile friendly MicroPython</em></dt>
+    <dd>
+    <p>Thanks to MicroPython in PyScript, there is a compelling story for
+    Python on mobile.</p>
+
+    <p>MicroPython is small and fast enough that your app will start quickly
+    on first load, and almost instantly (due to the cache) on subsequent
+    runs.</p></dd>
+
+    <dt><em>Parallel execution</em></dt>
+    <dd>Thanks to a browser technology called
+    <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API">web workers</a>
+    expensive and blocking computation can run somewhere other than the main
+    application thread that controls the user interface. When such work is done
+    on the main thread, the browser appears frozen. Web workers ensure
+    expensive blocking computation happens elsewhere. Think of workers as
+    independent subprocesses in your web page.</dd>
+
+    <dt><em>Rich and powerful plugins</em></dt>
+    <dd>
+    <p>As you'll see, PyScript has a small, efficient yet powerful core called
+    <a href="https://github.com/pyscript/polyscript">PolyScript</a>.</p>
+    <p>Most of the functionality of PyScript is actually implemented through
+    PolyScript's <a href="#plugins">plugin system</a>.</p>
+
+    <p>This approach means we get a clear separation of concerns: PolyScript
+    can focus on being small, efficient and powerful, whereas the PyScript
+    related plugins allow us to build upon the generic features provided by
+    PolyScript.</p>
+    <p>More importantly, because there is a plugin system, folks
+    <em>independent of the PyScript core team</em> have a way to create their
+    own plugins so we get a rich ecosystem of functionality that reflects the
+    unique and diverse needs of PyScript's users.</p>
+    </dd>
+</dl>
+
+## First steps
+
+It's simple:
+
+* tell your browser to use PyScript, then,
+* tell PyScript how to run your Python code.
+
+For the browser to use PyScript, simply add a `<script>` tag, whose `src`
+attribute references a CDN url for `pyscript.core`, to your HTML document's
+`<head>`. You may also add a reference to optional PyScript related CSS:
+
+**TODO: USE CORRECT CDN URL**:
+
+```html title="Reference PyScript in your HTML"
+<!doctype html>
+<html>
+    <head>
+        <!-- Recommended meta tags -->
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <!-- optional PyScript CSS -->
```

Comment:
I would remove `optional`. Better to teach people to always include it

Reply:
There have been many times when I've not needed the `pyscript.css` and used something else instead.

Reply:
yes, but the current `pyscript.css` is very minimal:
https://github.com/pyscript/pyscript/blob/a0bc4006cdd3eeb81a55460fb66dc089f5e08f90/pyscript.core/src/core.css#L1-L6

basically, it is needed to avoid FOUC, especially if you use `<py-script>` or `<py-config>`.
It is "optional" in the sense that your app still works without, but there is no good reason for teaching people not to use it, IMHO.

/cc @WebReflection since he cares a lot about FOUC :)